docs/.rubocop.yml

```diff
@@ -32,3 +32,7 @@ Style/RedundantRegexpArgument:
 # Want to be able to display partial formulae in the docs.
 Style/TopLevelMethodDefinition:
   Enabled: false
+
+# Formulae and Casks no longer use `rm_f`/`rm_rf`, so the docs need to match.
+Lint/NonAtomicFileOperation:
```

Comment:
Ideally we'd keep this around for `mkdir_p`.

Is this disabled/enabled in the other Homebrew `.rubocop.yml` file(s)? If so: should it be adjusted there too?

Reply:
It's enabled by default. But it only complained about this one when I removed the f.

Reply:
I think that's because this is a non-atomic operation now. Particularly in post-install where we don't fully control the location: this is potentially problematic.

See: https://docs.rubocop.org/rubocop/cops_lint.html#lintnonatomicfileoperation

I think having this still enabled for `mkdir_p` but also having this be consistent everywhere would be desirable if possible.